### ISSUE TITLE
Internal Documentation 404

### ISSUE THREAD

**Issue by MatanSilver** [marcobambini/gravity#230]
In the wiki, there is a link to the internal documentation which links to https://marcobambini.github.io/gravity/internals/index.html

This appears to be a non-existent file, and returns a 404 error.

**Comment by felipetesc** [comment]
Hello. As far as I understand, the documentation was changed in order to allow us to use the new web site. I believe the page on the wiki needs to be updated. You can read here(https://github.com/marcobambini/gravity/issues/221) to clarify what's happening when it comes about the documentation. I have read all docs before, and the internal's page didn't have any contents. My guess is that it's being written at this moment.

**Comment by marcobambini** [comment]
Thanks @MatanSilver I fixed the link.
@felipetesc is right, I plan to spend the next days improving existing user side documentation and finally improve the internals doc (not yet published). I would need help in the official documentation.

**Comment by MatanSilver** [comment]
I noticed that @hallzy wrote some good documentation for the map, filter, and reduce list functions that I wrote a while back. I think all that was missing documentation from those class methods was the sort(), sorted(), reverse(), and reversed() functions. Just an idea of a place to start that isn't really subject to personal preference on the details. That's also something you wouldn't need to read all the source code for, only the relevant parts of `gravity_core.c` (@felipetesc).
